lib/style.css

```diff
@@ -281,6 +281,11 @@ td.feed-source-info {
   box-shadow: none;
 }
 
+/* Exception date range date picker */
+.entity-details-body .input-group.date .input-group-addon {
```

Comment:
You might need an additional CSS class in `ExceptionDateRange` so that this does not apply to the existing date time picker.

Reply:
Yep this did need a class name to avoid styling the other date pickers, this should be fixed in https://github.com/ibi-group/datatools-ui/pull/839/commits/1f35f86ca4e64ac41fd21ae8f148a7da54efb222